stytch/cryptowallet/cryptowallet.go

```diff
@@ -50,6 +50,9 @@ func (c *Client) Authenticate(body *stytch.CryptoWalletAuthenticateParams,
 	return &retVal, err
 }
 
+// AuthenticateWithClaims fills in the claims pointer with custom claims from the response.
+// Pass in a map with the types of values you're expecting so that this function can marshal
+// the claims from the response. See session_test.go for an example
```

Comment:
It might be hard to find the part of `session_test.go` that's relevant. It might be nice to set up an [example](https://go.dev/blog/examples) that shows up nicely in the godoc.